script/tool/lib/src/publish_check_command.dart

```diff
@@ -26,9 +29,22 @@ class PublishCheckCommand extends PluginCommand {
           'the SDK constraint is a pre-release version, is ignored.',
       defaultsTo: false,
     );
+    argParser.addFlag(_logStatusFlag,
+        help:
+            'Logs the check-publish final status to a defined string as the last line of the command output.\n'
```

Comment:
Mixing output for humans and output for machines is generally not a good idea; I would rather we make a `--machine` flag that switches entirely to an output meant for parsing.